build.sbt

```diff
@@ -53,6 +55,10 @@ resolvers ++= Seq(
     Resolver.file("Local", file(Path.userHome.absolutePath + "/.ivy2/local"))(Resolver.ivyStylePatterns)
 )
 
+buildInfoPackage := "app"
+buildInfoOptions += BuildInfoOption.BuildTime
```

Comment:
You're right, it's not surfaced, and the line could be removed - I added it mainly because I thought it was nice to have a standard option for build time, which I had not seen before, and build time is surfaced on other Guardian projects like Ophan. But yep, could be removed!
```suggestion
```